Sources/ServiceLifecycle/ServiceGroup.swift

```diff
@@ -338,6 +338,7 @@ public actor ServiceGroup: Sendable {
                                 group: &group,
                                 gracefulShutdownManagers: gracefulShutdownManagers
                             )
+                            return .failure(error)
                         } catch {
                             return .failure(error)
```

Comment:
This is a different error, no?

Reply:
Yes this is the error if the group fails to shutdown. What do you expect here? The error from the original service that threw or the error during shutdown?

Reply:
Agreed. We are also logging that another service threw during shutdown and what error it threw.